grakn/rpc/transaction.py

```diff
@@ -135,14 +135,20 @@ def _fetch(self, request_id: str):
             try:
                 response = next(self._response_iterator)
             except grpc.RpcError as e:
+                print("Raising RpcError as GraknClientException")
                 self._transaction_was_closed = True
                 grakn_exception = GraknClientException(e.details())
                 for response_queue in self._response_queues.values():
                     response_queue.put(grakn_exception)
                 # noinspection PyUnresolvedReferences
                 raise grakn_exception
             except StopIteration:
+                print("Raising transaction closed as GraknClientException")
```

Comment:
@alexjpwalker is it a debug leftover?

Reply:
Ahhh I already fixed this locally but forgot to push, sorry!